autoload/which_key/util.vim

```diff
@@ -78,3 +78,26 @@ function! which_key#util#format(mapping) abort
   " let l:ret = substitute(l:ret, '^<Plug>', '', '')
   return l:ret
 endfunction
+
+function! s:m_char(char)
+  if a:char == '"'
+    return ["<M-\">", "\<M-\">"]
+  endif
+  let m_char = '<M-' . a:char . '>'
+  let m_char_code = eval('"\' . m_char . '"')
+  return [m_char, m_char_code]
+endfunction
+
+function! s:to_list(str)
+  return split(a:str, '\zs')
+endfunction
+
+
+let s:chars = s:to_list('AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz0123456789')
+      \ + s:to_list('`~!@#$%^&*()-_=+')+ s:to_list(' 	') + s:to_list('[]{};:''"\|,./<>?')
```

Comment:
To my knowledge, this is actually to get the [ASCII printable
characters](https://theasciicode.com.ar/), which could be simplified as `map(range(32, 126), 'nr2char(v:val)')`.